### PR TITLE
[RW-6398][risk=no] adding has_whole_genome_variant flag to CB search

### PR DESCRIPTION
**Pull request by freemabd**
This PR adds the has_whole_genome_variant flag to CB search. 
Also, updated the criteria table to include this in the search drop down menu as well as adding args to the circle ci job that builds the CDR indices.

### CODE REVIEW

**Review by jscherdin — CHANGES_REQUESTED** [review]
just one thing. otherwise looks good

File: api/db-cdr/generate-cdr/make-bq-criteria-tables.sh [hunk 1]
```diff
@@ -1449,6 +1449,36 @@ SELECT
...
+        , type
+        , name
+        , is_group
+        , is_selectable
+        , has_attribute
+        , has_hierarchy
+    )
+SELECT
+    (SELECT MAX(id) FROM \`$BQ_PROJECT.$BQ_DATASET.cb_criteria\`)+1 AS id
+    , 0
```
[the remark this answers is not in the document]
Comment (freemabd) [reply api/db-cdr/generate-cdr/make-bq-criteria-tables.sh:1472]: I just did exactly what was done for fitbit.

[the remark this answers is not in the document]
Comment (freemabd) [reply api/db-cdr/generate-cdr/make-bq-criteria-tables.sh]: Changed.

**Review by jscherdin — APPROVED** [review]
LGTM

**Pull request merged**